utils/release_notes.rst.in

```diff
@@ -0,0 +1,30 @@
+==================================
+Cyclus @VERSION@ Release Notes
+==================================
+:Author: Cyclus Developers
+:Website: http://fuelcycle.org/
+:Contact: https://groups.google.com/forum/#!forum/cyclus-dev
+
+The Cyclus core team is excited to announce the latest stable version of the
+cyclus ecosystem!
+
+Major features
+==============
+!--- TODO: Fill in major features --!
```

Comment:
Exactly.  These lines should never be rendered.  If you want something to be rendered, I think a warning directive would be much better / flashier.